lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}
+
+# Define the location of your websockets here
+<%= application_name %>.cable = Cable.createConsumer 'ws://localhost:28080'

```

Comment:
When would the protocol ever be anything but `was` or `ws`? Seems like `createConsumer` should just set that up:

```CoffeeScript
<%= application_name %>.cable = Cable.createConsumer 'localhost:28080'
```

Perhaps a `protocol: true` argument would make it clearer that it's automatically handled. Don't know if better documentation is enough.